R/estimate_infections.R

```diff
@@ -695,17 +707,20 @@ format_fit <- function(posterior_samples, horizon, shift, burn_in, start_date,
     format_out$samples <- format_out$samples[, strat := NA]
   }
   # add type based on horizon
+  # nolint start
   format_out$samples <- format_out$samples[
     ,
-    type := data.table::fcase(
+    type := data.table::fifelse(
       date > (max(date, na.rm = TRUE) - horizon),
       "forecast",
-      date > (max(date, na.rm = TRUE) - horizon - shift),
-      "estimate based on partial data",
-      is.na(date), NA_character_,
-      default = "estimate"
+      data.table::fifelse(
+        date > (max(date, na.rm = TRUE) - horizon - shift),
+        "estimate based on partial data",
+        "estimate"
       )
+    )
   ]
+  # nolint end
```

Comment:
unrelated - left over from merging/rebasing `main`?

Reply:
Fixed in https://github.com/epiforecasts/EpiNow2/pull/363/commits/967d3cdb29b71daff6488d2c1529f53fb56e35f5